2nd-gen/uppercase-firestore/functions/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2023 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+// [START all]
+// [START import]
+// The Cloud Functions for Firebase SDK to create Cloud Functions and set up triggers.
+const {logger} = require('firebase-functions');
+const {onRequest} = require('firebase-functions/v2/https');
+const {onDocumentCreated} = require('firebase-functions/v2/firestore');
+
+// The Firebase Admin SDK to access Firestore.
+const admin = require('firebase-admin');
```

Comment:
Can you please use the [modular import syntax for the admin SDK](https://firebase.google.com/docs/admin/migrate-node-v10#using-v10-modular-entry-points)? I'd like to start recommending that going forward.

```suggestion
const { initializeApp } = require('firebase-admin/app');
const { getFirestore } = require('firebase-admin/firestore');
```

Note: this will also require code changes below